src/mono/sample/wasm/Directory.Build.props

```diff
@@ -6,6 +6,7 @@
     <TargetOS>browser</TargetOS>
     <TargetArchitecture>wasm</TargetArchitecture>
     <RuntimeIdentifier>browser-wasm</RuntimeIdentifier>
+    <MonoWasmBuildVariant Condition="'$(WasmEnableThreads)' == 'true'">multithread</MonoWasmBuildVariant>
```

Comment:
This doesn't make sense to me.  the sample apps are like user apps.  but `MonoWasmBuildVariant` is a property of how we build the runtime.  By the time any of this code runs, the runtime is already built. How can setting it here have any effect?